src/App.tsx

```diff
@@ -1,98 +1,22 @@
-import OrgCreationPage from 'screens/OrgCreationPage/OrgCreationPage';
-import SelectOrganizationPage from 'screens/SelectOrganization/SelectOrganization';
-function App(): JSX.Element {
-  const isLoggedInAs = localStorage.getItem('isloggedinas');
+import OrganizationDashboard from 'screens/OrganizationDashboard/OrganizationDashboard';
 
+function App(): JSX.Element {
+  const isLoggedIn = localStorage.getItem('IsLoggedIn');
   return (
     <>
       <Switch>
-        <Route exact path="/" component={WelcomePage} />
-        <Route exact path="/login" component={LoginPage} />
-        <Route path="/orghome">
-          {isLoggedInAs != 'ORGADMIN' ? (
-            <Redirect to="/login" />
-          ) : (
-            <OrgAdminHomePage />
-          )}
-        </Route>
-        <Route path="/orgmember">
-          {isLoggedInAs != 'ORGADMIN' ? (
-            <Redirect to="/login" />
-          ) : (
-            <OrgAdminMemberPage />
-          )}
-        </Route>
-        <Route path="/selectorg">
-          {isLoggedInAs != 'ORGADMIN' ? (
-            <Redirect to="/login" />
-          ) : (
-            <SelectOrganizationPage />
-          )}
-        </Route>
-        <Route exact path="/supermember">
-          {isLoggedInAs != 'SUPERADMIN' ? (
-            <Redirect to="/login" />
-          ) : (
-            <SuperAdminMemberPage />
-          )}
-        </Route>
-        <Route exact path="/superorg">
-          {isLoggedInAs != 'SUPERADMIN' ? (
-            <Redirect to="/login" />
-          ) : (
-            <SuperAdminOrgPage />
-          )}
-        </Route>
-        <Route exact path="/superdash">
-          {isLoggedInAs != 'SUPERADMIN' ? (
-            <Redirect to="/login" />
-          ) : (
-            <SuperAdminDashboard />
-          )}
-        </Route>
-        <Route path="/superorgmember">
-          {isLoggedInAs != 'SUPERADMIN' ? (
-            <Redirect to="/login" />
-          ) : (
-            <SuperAdminOrgMemberPage />
-          )}
-        </Route>
-        <Route exact path="/superuserorg">
-          {isLoggedInAs != 'SUPERADMIN' ? (
-            <Redirect to="/login" />
-          ) : (
-            <SuperAdminUserOrgPage />
-          )}
-        </Route>
-        <Route path="/superorghome">
-          {isLoggedInAs != 'SUPERADMIN' ? (
-            <Redirect to="/login" />
-          ) : (
-            <SuperAdminOrgHomePage />
-          )}
-        </Route>
-        <Route path="/createorg">
-          {isLoggedInAs != 'SUPERADMIN' ? (
-            <Redirect to="/login" />
-          ) : (
-            <OrgCreationPage />
-          )}
-        </Route>
+        <Route exact path="/" component={LoginPage} />
+        {isLoggedIn == 'TRUE' ? (
+          <Route path="/orgdash" component={OrganizationDashboard} />
+        ) : null}
+        {isLoggedIn == 'TRUE' ? (
+          <Route path="/orgpeople" component={OrganizationPeople} />
+        ) : null}
```

Comment:
You can replace this with

```suggestion
        <Route exact path="/" component={LoginPage} />
        {isLoggedIn == 'TRUE' ? (
          <Route path="/orgdash" component={OrganizationDashboard} />
          <Route path="/orgpeople" component={OrganizationPeople} />
        ) : null }
```
Since the same conditional is being applied to both routes